test/ffprobe_test.go

```diff
@@ -0,0 +1,50 @@
+package ffprobe
```

Comment:
Renaming the package also implied moving the file. As a general rule the `_test.go` exists in the same package and directory of the thing you are testing.

Reply:
Hi, what's your meaning here? so what's the package name should be?

Reply:
No the package name is fine, I meant to move the package one directory up into the root. So it sits next to the `ffprobe.go` file it is testing.

Reply:
If you want I can merge it and make the change myself :)

Reply:
sure, you can  :)